dev/Repeater/ItemsSourceView.h

```diff
@@ -22,6 +22,8 @@ class ItemsSourceView :
     void CollectionChanged(winrt::event_token const& token);
 #pragma endregion
 
+    int IndexOf(winrt::IInspectable const& value);
```

Comment:
This can move into the region above and while we are here can we change it from IDataSource to ItemsSourceView in the pragma.